https://build.opensuse.org/package/view_file/GNOME:Next/gtk3/gtk3-boo1121456-a11y-export.patch

From 3313213d61b2fdca96230223393c1c5054b2de49 Mon Sep 17 00:00:00 2001
From: Mike Gorse <mgorse@alum.wpi.edu>
Date: Fri, 11 Jan 2019 16:21:21 -0600
Subject: [PATCH] A11y: export
 gtk_cell_accessible_parent_get_(row|column)_header_cells

These functions were missing GDK_AVAILABLE_IN macros, so they were
listed in the gir but not exported.
---
 gtk/a11y/gtkcellaccessibleparent.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gtk/a11y/gtkcellaccessibleparent.h b/gtk/a11y/gtkcellaccessibleparent.h
index 8406574dcb..5e99a28c56 100644
--- a/gtk/a11y/gtkcellaccessibleparent.h
+++ b/gtk/a11y/gtkcellaccessibleparent.h
@@ -128,10 +128,10 @@ void     gtk_cell_accessible_parent_get_cell_position(GtkCellAccessibleParent *p
                                                       GtkCellAccessible       *cell,
                                                       gint                    *row,
                                                       gint                    *column);
-
+GDK_AVAILABLE_IN_3_24
 GPtrArray   *gtk_cell_accessible_parent_get_column_header_cells (GtkCellAccessibleParent *parent,
                                                                  GtkCellAccessible       *cell);
-
+GDK_AVAILABLE_IN_3_24
 GPtrArray   *gtk_cell_accessible_parent_get_row_header_cells    (GtkCellAccessibleParent *parent,
                                                                  GtkCellAccessible       *cell);
 
-- 
2.18.0

